onnx/common/path.cc

```diff
@@ -0,0 +1,19 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// Copyright (c) ONNX Project Contributors.
+// Licensed under the MIT license.
+
+#include "onnx/common/path.h"
+
+namespace ONNX_NAMESPACE {
+
+std::string path_join(std::string origin, std::string append) {
```

Comment:
why are you making a copy of origin string and the one to append. This should be const ref

Reply:
Makes sense. Updated.